diff -aurN hydrogen-music-hydrogen-def9a33.orig/src/core/CMakeLists.txt hydrogen-music-hydrogen-def9a33/src/core/CMakeLists.txt
--- hydrogen-music-hydrogen-def9a33.orig/src/core/CMakeLists.txt	2012-10-22 11:21:42.929992268 +0200
+++ hydrogen-music-hydrogen-def9a33/src/core/CMakeLists.txt	2012-10-22 11:39:24.988815514 +0200
@@ -44,6 +44,7 @@
     ${LASH_LIBRARIES}
     ${LRDF_LIBRARIES}
     ${RUBBERBAND_LIBRARIES}
+    pthread
 )
 
 #SET_TARGET_PROPERTIES(hydrogen-core-${VERSION} PROPERTIES PUBLIC_HEADER   "${hydrogen_INCLUDES}" )
